lightly_solution.ipynb

```diff
@@ -126,7 +126,7 @@
     "# If you have a Nvidia GPU, add the flag to use it: --gpus all\n",
     "\n",
     "from pathlib import Path\n",
-    "absolute_dataset_path = Path(dataset_path).absolute()\n",
+    "absolute_dataset_path = (Path(dataset_path) / \"images\").absolute()\n",
```

Comment:
I think we should change this part of the notebook instead:
```
###### CHANGE THESE 2 VARIABLES
LIGHTLY_TOKEN = "CHANGE_ME_TO_YOUR_TOKEN"  # Copy from https://app.lightly.ai/preferences
DATASET_PATH = Path("CHANGE_ME")  # e.g., Path("/path/to/images") or Path("clothing_dataset")
######
```
And set `Path("clothing_dataset/images")` there. Otherwise users might set their own dataset path and then we add an "images" subdir to their custom path.